test/app/src/configArea.ts

```diff
@@ -95,7 +95,7 @@ export default class ConfigArea {
   }
 
   addEventListeners(): void {
-    this.configForm.addEventListener('change', (event) => {
```

Comment:
nit: enable ESLINT to error out for unused variable?

Reply:
sure, we can enable more lint on the test app. It's pretty new so can use some more love.

Reply:
It was coming as a warning in the test app, whereas this is an error with widget source. Seems like a quirk with Typescript: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-unused-vars.md

I'll get it fixed so it is consistently returning an error